packages/openapi-specification/openapi.yaml

```diff
@@ -1,742 +1,177 @@
+          description: Invalid ID
         '404':
-          description: User not found
+          description: Record not found
       security:
-        - api_key: []
-externalDocs:
-  description: Find out more about Swagger
-  url: 'http://swagger.io'
+        - records_auth:
+          - write:records
+          - read:records
+
+
 components:
-  requestBodies:
-    UserArray:
-      content:
-        application/json:
-          schema:
-            type: array
-            items:
-              $ref: '#/components/schemas/User'
-      description: List of user object
-      required: true
-    Pet:
-      content:
-        application/json:
-          schema:
-            $ref: '#/components/schemas/Pet'
-        application/xml:
-          schema:
-            $ref: '#/components/schemas/Pet'
-      description: Pet object that needs to be added to the store
-      required: true
   securitySchemes:
-    petstore_auth:
+    records_auth:
       type: oauth2
       flows:
         implicit:
-          authorizationUrl: 'http://petstore.swagger.io/api/oauth/dialog'
+          authorizationUrl: ''
           scopes:
-            'write:pets': modify pets in your account
-            'read:pets': read your pets
-    api_key:
-      type: apiKey
-      name: api_key
-      in: header
+            write:records: add and modify records in the website records list
+            read:records: read records from the website records list
+
+  requestBodies:
+    Record:
+      description: A record object
+      required: true
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/Record'
+
   schemas:
-    Order:
-      title: Pet Order
-      description: An order for a pets from the pet store
-      type: object
-      properties:
-        id:
-          type: integer
-          format: int64
-        petId:
-          type: integer
-          format: int64
-        quantity:
-          type: integer
-          format: int32
-        shipDate:
-          type: string
-          format: date-time
-        status:
-          type: string
-          description: Order Status
-          enum:
-            - placed
-            - approved
-            - delivered
-        complete:
-          type: boolean
-          default: false
-      xml:
-        name: Order
-    Category:
-      title: Pet category
-      description: A category for a pet
-      type: object
-      properties:
-        id:
-          type: integer
-          format: int64
-        name:
-          type: string
-          pattern: '^[a-zA-Z0-9]+[a-zA-Z0-9\.\-_]*[a-zA-Z0-9]+$'
-      xml:
-        name: Category
-    User:
-      title: a User
-      description: A User who is purchasing from the pet store
+    Record:
+      title: Website Record
+      description: A record of a website crawl
       type: object
+      required:
+        - url
+        - boundaryRegEx
       properties:
```

Comment:
Maybe we also want the time of last execution here

Reply:
I added time and status of last execution, but I don't know if they should be required, since before the first execution they will be empty

Reply:
just so you know, Viki added the comment to line 146 (i.e. `properties`) - GitHub just always shows the 4 lines above it for some reason. Confused me quite a lot when I started with it 😅